pg_backup_api/version.txt

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
This can probably go to 1.0.0 now, since we've stabilized the spec version to 1.0.0.

Reply:
Yes I think you're right - we can update the version when we cut the first release.